java/src/edu/wpi/first/wpilibj/networktables/NetworkTable.java

```diff
@@ -852,8 +852,8 @@ public boolean putRaw(String key, ByteBuffer value, int len) {
   public boolean putValue(String key, Object value) throws IllegalArgumentException {
     if (value instanceof Boolean)
       return NetworkTablesJNI.putBoolean(pathWithSep + key, ((Boolean)value).booleanValue());
-    else if (value instanceof Double)
-      return NetworkTablesJNI.putDouble(pathWithSep + key, ((Double)value).doubleValue());
+    else if (value instanceof Number && !(value instanceof Long))
```

Comment:
I'm not sure which is correct, but we need to either create a whitelist or allow everything in. And I  _really_ don't like the idea of a whitelist.